doc/rst/technotes/driver.rst

```diff
@@ -0,0 +1,82 @@
+.. _readme-driver:
+
+====================
+Compiler Driver Mode
+====================
+
+.. note::
+
+   The compiler driver mode is considered experimental, and is not yet
+   well-tested for correctness or performance.
+   If you use it and encounter a bug or limitation not yet documented as a
+   `Github issue <https://github.com/chapel-lang/chapel/issues>`_, consider
+   filing an issue as described in :ref:`readme-bugs`.
+
+The Chapel compiler ``chpl`` currently runs as a single executable responsible
+for compilation, assembly, and linking, with some components done in
+subprocesses depending on which backend is used. By contrast, many compiler
+executables (including ``gcc`` and ``clang``) are actually compiler drivers, a
+smaller driver program which is responsible for processing arguments and
```

Comment:
"compiler driver**s**" is plural, but the subsequent definition ("a smaller driver program") is singular. How about:

> By contrast, many compiler executables (including `gcc` and `clang`) are actually _compiler drivers_. A compiler driver is a smaller...